lib/client/conntest/database/mysql.go

```diff
@@ -72,7 +72,7 @@ func (p *MySQLPinger) Ping(ctx context.Context, params PingParams) error {
 	}
 
 	defer func() {
-		if err := conn.Close(); err != nil {
+		if err := conn.Quit(); err != nil {
```

Comment:
Our old [patch](https://github.com/gravitational/go-mysql/pull/3/files) sends `COM_QUIT` in `conn.Close`. Now the driver provides a `Quit` function that does a similar thing so switching to `Quit` where I can.